syz-cluster/pkg/triage/diff.go

```diff
@@ -0,0 +1,49 @@
+// Copyright 2025 syzkaller project authors. All rights reserved.
+// Use of this source code is governed by Apache 2 LICENSE that can be found in the LICENSE file.
+
+package triage
+
+import (
+	"path/filepath"
+
+	"github.com/google/syzkaller/pkg/vcs"
+	"github.com/google/syzkaller/syz-cluster/pkg/api"
+)
+
+// For now, follow the simplest strategy - just verify whether the series modifies only the files
```

Comment:
Add a TODO that this should be replaced with build signatures, rather than extended be more and more complex.